engines/access/metaengine.cpp

```diff
@@ -178,6 +185,130 @@ SaveStateDescriptor AccessMetaEngine::querySaveMetaInfos(const char *target, int
 	return SaveStateDescriptor();
 }
 
+Common::KeymapArray AccessMetaEngine::initKeymaps(const char *target) const {
+	using namespace Common;
+	using namespace Access;
+
+	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "access-default", _("Default keymappings"));
+
+	Action *act;
+
+	act = new Action(kStandardActionLeftClick, _("Move / Interact / Skip"));
+	act->setLeftClickEvent();
+	act->addDefaultInputMapping("MOUSE_LEFT");
+	act->addDefaultInputMapping("JOY_A");
+	engineKeyMap->addAction(act);
+
+	act = new Action(kStandardActionRightClick, _("Skip"));
+	act->setRightClickEvent();
+	act->addDefaultInputMapping("MOUSE_RIGHT");
+	act->addDefaultInputMapping("JOY_B");
+	engineKeyMap->addAction(act);
+
+	act = new Action("PAUSE", _("General Main Menu"));
+	act->setCustomEngineActionEvent(kActionPause);
+	act->addDefaultInputMapping("F10");
+	act->addDefaultInputMapping("JOY_START");
+	engineKeyMap->addAction(act);
+
+	act = new Action("SKIP", _("Skip Movie"));
```

Comment:
I had an impression that we already fixed all to the Sentence Case, no? Why are you reintroducing Title Case over all of these actions...